docs/section11/Rule_11-11.md

```diff
@@ -0,0 +1,58 @@
+# Service_Water_Heating - Rule 11-11  
+**Schema Version:** 0.0.36  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-11  
+
+**Rule Description:** For buildings that will have no service water-heating loads, no service water-heating shall be modeled in baseline building model
+
+**Rule Assertion:** Options are PASS/FAIL
+
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, c
+
+**Evaluation Context:** B-RMD each building segment
+**Data Lookup:**   
+
+**Function Call:** 
+- **get_component_by_id**  
+- **get_SWH_equipment_associated_with_each_SWH_bat**  
+- **get_SWH_types_and_SWH_use**  
+- **get_SWH_uses_associated_with_each_building_segment**  
+
+**Applicability Checks:**
+
+- look at each building segment in the proposed model: `for building_segment in B_RMD.building.building_segments:`
+    - create a boolean has_swh_loads and set it to false: `has_swh_loads = false`
+    - get the service water heating uses in the proposed building segment `service_water_heating_use_ids = get_SWH_uses_associated_with_each_building_segment(P_RMD, building_segment.id)`
+    - look at each service water heating use id: `for swh_use_id in service_water_heating_use_ids:`
+        - get the swh_use using get_component_by_ID: `swh_use = get_component_by_ID(P_RMD, swh_use_id)`
+        - check to see if the swh use is greater than 0 (ie there is any amount of hot water used): `if swh_use.use > 0:`
```

Comment:
I know you have been checking if the .use is >0 to see if there are SWH loads but wonder if it is also necessary to make sure the schedule is not 0 for every hour. Maybe that is overkill. I don't feel strongly.

Reply:
I think that's overkill